docs/content/install/install-on-gae.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Install on Google App Engine"
+date: 2019-09-27T17:48:40+10:00
+draft: false
+weight: 4
+---
+
+
+[Google App Engine (GAE)](https://cloud.google.com/appengine/) is a Google service allows applications to be deployed without provisioning the underlying hardware. It is similar to Azure Container Engine which is covered in a [previous section](/install/install-on-aci). This guide will demonstrate how you can get Athens running on GAE.
+
+## Selecting a Storage Provider
+
+Athens currently supports a number of storage drivers. For quick and easy use on GAE, we recommend using the local disk provider. For more permanent use, we recommend using MongoDB or other more persistent infrastructure. For other providers, please see the [storage provider documentation](/configuration/storage).
```

Comment:
@schafer14 would it make sense to use GCS for storage on app engine? Not sure if that's quick and easy enough. What do you think?

Reply:
It will be very unusual to use Appengine with something other than GCS.

Reply:
That does sound good. I'll do that.

Reply:
To both config/gae and to gcs.

Reply:
@arschles would the scripts directory be a better place for these files then the config file? I don't have a preference, I am just wondering if that is another option.